api/agent.go

```diff
@@ -730,6 +730,20 @@ func (a *Agent) ForceLeave(node string) error {
 	return nil
 }
 
+//ForceLeavePrune is used to have an a failed agent removed
+//from the list of members
+func (a *Agent) ForceLeavePrune(node string) error {
+	r := a.c.newRequest("PUT", "/v1/agent/force-leave/"+node)
+	q := &QueryOptions{Prune: true}
```

Comment:
r.params.Set("compile-dc", opts.EvaluateInDatacenter)
```suggestion
	q := &QueryOptions{Prune: true}
	r.params.Set("prune", "1")
```